src/postgres/tree/PostgresFunctionTreeItem.ts

```diff
@@ -24,13 +25,14 @@ export class PostgresFunctionTreeItem extends AzureTreeItem<ISubscriptionContext
         super(parent);
         this.schema = row.schema;
         this.name = row.name;
+        this.args = row.args;
         this.id = String(row.oid);
         this.definition = row.definition;
         this.isDuplicate = isDuplicate;
     }
 
     public get label(): string {
-        return this.name;
+        return `${this.name}(${this.args})`;
```

Comment:
I'm skeptical that this is common enough to be worth it. Even if there are duplicates, there's probably not that many and I can just click the function to open it up and view the args

Reply:
I agree this is a pretty unlikely case. Updating the label is probably overkill, but I think it'll be worthwhile to add the functions's arguments to the `PostgresFunctionTreeItem` id once https://github.com/microsoft/vscode-cosmosdb/pull/1420 is merged. Otherwise, loading functions with the same name & different args will lead to this error:

<img width="466" alt="Screen Shot 2020-04-22 at 12 42 20 PM" src="https://user-images.githubusercontent.com/22795803/80027010-f09d7300-8497-11ea-90e1-865aeed47d24.png">

Reply:
Well to @nturinski's [point](https://github.com/microsoft/vscode-cosmosdb/pull/1420#discussion_r409911922) - that's why oid is preferred as the id. I think it's the only reliable way to uniquely identify a function

Reply:
True - I'll add a query to get the oid after creating the function tree item. It'll prevent any more issues like this one